test/unit/tile.js

```diff
@@ -91,6 +91,39 @@ const assertGoogleTiles = function (directory, expectedTileSize, expectedLevels,
   });
 };
 
+// Verifies all square tiles in a given output directory are > size with overlap
+const assertTileOverlap = function (directory, tileSize) {
+  // Get levels
+  const levels = fs.readdirSync(directory);
+  // Get tiles
+  const tiles = [];
+  levels.forEach(function (level) {
+    // Verify level directory name
+    assert.strictEqual(true, /^[0-9]+$/.test(level));
+    fs.readdirSync(path.join(directory, level)).forEach(function (tile) {
+      // Verify tile file name
+      assert.strictEqual(true, /^[0-9]+_[0-9]+\.jpeg$/.test(tile));
+      tiles.push(path.join(directory, level, tile));
+    });
+  });
+
+  eachLimit(tiles, 8, function (tile, done) {
+    sharp(tile).metadata(function (err, metadata) {
+      if (err) {
+        done(err);
+      } else {
+        // Only checks square tiles
+        if (metadata.width >= tileSize && metadata.height >= tileSize) {
```

Comment:
Is there a way to verify the presence of overlap without introducing a conditional statement into the unit tests? Perhaps we could check only the penultimate level, the images in which should all be square and should all meet this condition.

Reply:
I've adjusted the test case in question. Now the test will choose a square tile from the approximate center of the image in the outermost tile level. Thanks for the feedback and let me know what you think.